components/widgets/climate/emissions-deforestation-drivers/selectors.js

```diff
@@ -241,9 +243,41 @@ export const parseTitle = createSelector(
   }
 );
 
+export const parseAlerts = createSelector(
+  [getAlerts, getLocationName, getAdm0],
+  (alerts, locationLabel, adm0) => {
+    const countriesWithNewWarningText = [
```

Comment:
I suggest to change the const name just to give more context like `countriesWithOutdatedData` or something like that.